src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle {
+    double kat1;
+    double kat2;

```

Comment:
Similarly, 'kat2' is not descriptive. Consider renaming it to 'leg2' or something more meaningful.